apps/web/components/eventtype/EventAppsTab.tsx

```diff
@@ -133,12 +136,28 @@ export const EventAppsTab = ({ eventType }: { eventType: EventType }) => {
     <>
       <div>
         <div className="before:border-0">
-          {isManagedEventType && (
+          {(isManagedEventType || isChildrenManagedEventType) && (
             <Alert
-              severity="neutral"
+              severity={appsDisableProps.isLocked ? "neutral" : "green"}
               className="mb-2"
-              title={t("locked_for_members")}
-              message={t("locked_apps_description")}
+              title={
+                <Trans i18nKey={`${lockedText}_${isManagedEventType ? "for_members" : "by_team_admins"}`}>
+                  {lockedText[0].toUpperCase()}
+                  {lockedText.slice(1)} {isManagedEventType ? "for members" : "by team admins"}
+                </Trans>
+              }
+              actions={<div className="flex h-full items-center">{appsDisableProps.LockedIcon}</div>}
```

Comment:
If enabling/disabling one toggle on this page all toggles changes
<img width="999" alt="Screenshot 2024-03-01 at 3 31 50 PM" src="https://github.com/calcom/cal.com/assets/30310907/53faad89-467f-4a5c-81b4-3bc1c8a944ff">

Reply:
This is too granular for now, so only added the locked/unlocked for All apps.